src/run.test.js

```diff
@@ -7,7 +7,7 @@ jest.spyOn(core, 'getInput')
 jest.spyOn(core, 'setFailed')
 jest.spyOn(core, 'setOutput')
 
-const index = require('./index')
+const run = require('./run').run
```

Comment:
Not related to this test, but any thoughts about changing the name of this file from `run.js` to something else? Maybe something like `main.js`.

My main issue is with the file name being the same as the method `run.run()`. Where something like `main.run()` might be cleaner.

Reply:
I believe this patch here addresses the issue reasonably. https://github.com/mdb/action-conventional-versioning/pull/1/files#diff-ded7d82adc6eea35c2e80c2cf91ea002bd00db3073decc2812add012d5872efcR6 Any thoughts?

Reply:
Nice! Removing the `run.run` stutter is definitely an improvement!